wasm/Driver.cpp

```diff
@@ -769,12 +769,22 @@ void LinkerDriver::link(ArrayRef<const char *> argsArr) {
             config->entry);
   }
 
-  // Is the entry symbol found
   for (ObjFile *file : symtab->objectFiles) {
     for (Symbol *sym : file->getSymbols()) {
+       // Is the entry symbol found
        if (toString(*sym) == config->entry) {
           symtab->entryIsUndefined = false;
        }
+
+       // Is sync_call entry symbol found
+       if (toString(*sym) == "sync_call") { // OK to hard code sync call entry here. Spring validate the entry function name to be "sync_call"
+          symtab->syncCallEntryIsUndefined = false;
+       }
+
+       if (symtab->entryIsUndefined && symtab->syncCallEntryIsUndefined) {
```

Comment:
Should this be `!` on both of these?

Reply:
https://github.com/AntelopeIO/cdt-lld/pull/2/commits/e59ac252d3caf05c5da2026a91abf11da44b0156